code/modules/mob/living/carbon/carbon.dm

```diff
@@ -637,6 +637,7 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 		throw_icon.icon_state = "act_throw_off"
 
 /mob/living/carbon/proc/throw_mode_on()
+	SIGNAL_HANDLER
```

Comment:
Would you add a comment as to why this is here.

Reply:
Sure